website/content/en/docs/about/under-the-hood/guarantees.md

```diff
@@ -11,6 +11,33 @@ Here you can find an overview of delivery guarantee types and their meaning as w
 the stability of our components. Next, you can head over to the [components] page and use filters to
 see which components support specific guarantees.
 
+## Acknowledgement guarantees
+
+Vector supports end-to-end acknowledgement for the majority of its
+sources and sinks. This is a system which tracks the delivery status of
+an event through the lifetime of that event as it travels from the
+originating source to any number of destination sinks. Support for this
+feature is indicated by the "acknowledgements" badge at the top of the
+relevant sink or source configuration reference page.
```

Comment:
I'm not actually seeing the badges. I think we may need to update `website/layouts/partials/docs/component-under-hero.html`?

Reply:
Ah, yes, I had been looking for that, but understood from another comment that the badges were found elsewhere. Fixed in e4878bf5f (I hope)

Reply:
👍 I see the badges now.